packages/pulumi/src/executors/up/up.impl.ts

```diff
@@ -24,7 +27,10 @@ export default async function creatExecutor(
       'pulumi up', 
       options.stack && `--stack=${options.stack}`,
       options.skipPreview && '--skip-preview',
-      options.yes && '--yes'
+      options.yes && '--yes',
+      options.suppressOutputs && '--suppress-outputs',
+      options.debug && '--debug',
```

Comment:
Quesing that `--debug` logs more? If so can we change this to the constant `USE_VERBOSE_LOGGING` from `@nx-extend/core`

Reply:
That's a great idea! Unfortunately, I am pressed on time these days. Feel free to slip that in there too if you don't want to wait ❤️

Reply:
Thanks, added these change to #139.